src/Jobs/ProcessAnnotatedFile.php

```diff
@@ -420,7 +420,7 @@ protected function getEllipseSvgAnnotation(array $tuples): SVGEllipse
     protected function computeRotationAngle(array $v, array $u): float
     {
         // If (upper) left and (upper) right coordinate have equal y coordinate, then there is no rotation
-        if ($v[1] === 0) {
+        if ($v[1] == 0) {
```

Comment:
Can you please explain why this fixes the error? The smallest changes are often the ones requiring most explanation :wink: 

My intuition would be to to a type conversion somewhere rather than to relax this check.

Reply:
Ok I think I got it. If all coordinates are equal you get `0.0 !== 0` here. Then I prefer `(intval($v[1]) === 0)`. I'll update this myself.

Reply:
In this array I store the vector from the right to the left point (difference of them). The coordinates of a annotation are saved as floats, which is why the differences are also saved as floats in `$v`. I could also have written `if ($v[1] === 0.0) {`, which does the same, I think.

It fixes the error, because I compare the (upper) right and (upper) left point. If they have the same y coordinate (no rotation), their difference is zero, which is stored in `$v[1]`. A point with equal coordinates would also have no difference in their y-coordinates of their "right" and "left" point.